src/LaunchDarklyProvider.ts

```diff
@@ -69,10 +69,10 @@ export default class LaunchDarklyProvider implements Provider {
     try {
       this.client = init(sdkKey, {
         ...options,
-        wrapperName: 'open-feature/node-server',
```

Comment:
When we form the wrapper header we combine the name and version with a `/`. So, just to prevent confusion, I am making this all one thing.